azure/templates/pds-pytest-karate-sandbox.yml

```diff
@@ -0,0 +1,21 @@
+steps:
+
+  - bash: poetry install
+    workingDirectory: $(Pipeline.Workspace)/s/$(SERVICE_NAME)/$(SERVICE_ARTIFACT_NAME)
+    displayName: Install poetry dependencies
+
+  - bash: |
+      cd karate-tests
+      bash build-sandbox.sh
+      docker run -d --name karate-sandbox -p 9090:9090 nhs/pds-sandbox:latest 
+      export APIGEE_ENVIRONMENT=karate
+      cd ..
+      poetry run pytest -v tests/sandbox/test_sandbox.py --junitxml=tests/sandbox-test-report.xml
+    displayName: Run pytest sandbox tests against Karate sandbox
+    workingDirectory: "$(Pipeline.Workspace)/s/$(SERVICE_NAME)/$(SERVICE_ARTIFACT_NAME)"
+
+  - task: PublishTestResults@2
+    displayName: 'Publish sandbox test results'
+    inputs:
+      testResultsFiles: $(Pipeline.Workspace)/s/$(SERVICE_NAME)/$(SERVICE_ARTIFACT_NAME)/tests/sandbox-test-report.xml
```

Comment:
would it be useful to date/time stamp the file so data for multiple runs can be accommodated rather than overwritten?

Reply:
But each run is persisted as an ADO run, so what use case are you thinking of?